benchmark_runner/common/template_operations/templates/hammerdb/internal_data/mariadb_template.yaml

```diff
@@ -109,20 +150,7 @@ spec:
         - name: mariadb-persistent-storage
```

Comment:
Is mariadb-persistent-storage volume not needed for LSO mode?

Reply:
You right, I run LSO always with odf_pvc == True.
I have fixed it in all the database:
{%- if storage_type == 'lso' or odf_pvc == True %}